data/wanderer/wanderers start.txt

```diff
@@ -2727,13 +2727,39 @@ mission "Wanderers Ap'arak 3: Pug"
 	source "Varu Tev'kei"
 	to offer
 		has "Wanderers Ap'arak 2: done"
-	npc kill
+	npc save
 		personality staying unconstrained uninterested
 		government "Pug (Wanderer)"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
+	to complete
+		never
+
+mission "Wanderers Ap'arak 3: Pug 2"
+	landing
+	invisible
+	source "Varu Tev'kei"
+	to offer
+		has "Wanderers Ap'arak 2: done"
+		not "Wanderers Ap'arak 3: done"
+	npc save
+		personality staying unconstrained uninterested
+		government "Pug (Wanderer)"
 		ship "Pug Arfecta" "Graak Sput Kchoo"
-	on complete
-		dialog `Shortly after defeating the Pug ships, you receive a message from Sobari Tele'ek: "We are [aware? incredulous?] of reports that you have [exterminated, eradicated] our Keepers in the Ap'arak system. This is unacceptable! From now on, you are no longer welcome in our [territory, space]."`
+	to complete
+		never
+
+mission "Wanderers Pug Hostile"
+	landing
+	invisible
+	source
+		government "Wanderer"
+	to offer
+		has "Wanderers Ap'arak 3: Pug: failed"
+		has "Wanderers Ap'arak 3: Pug 2: failed"
+	on offer
+		conversation
+			`Having defeated the Pug ships guarding Wanderer space, you receive a message from Sobari Tele'ek: "We are [aware? incredulous?] of reports that you have [exterminated, eradicated] our Keepers in the Ap'arak system. This is unacceptable! From now on, you are no longer welcome in our [territory, space]."`
```

Comment:
The opening here ("Having defeated..., you receive") is awkward and should be rephrased. I'd argue the text flows better if we drop the lead-in entirely, so we can explore a text structure that makes the message seem more like a formal letter from a Wanderer ambassador.

Reply:
Part of that line is because the original is "Shortly after defeating the Pug ships, you receive a message...", which sounded fine. The other part is someone could jump away from the system, or even leave Wanderer space entirely before coming back months later. That preamble helps remind a player of what they did from the jump, and dropping it entirely feels really short.

Reply:
Yeah, just identifying some awkward phrasing. I didn't have any better alternatives, which is why I suggested perhaps changing the format to be totally different.